src/interfaces/GeoChart.ts

```diff
@@ -35,7 +35,7 @@ export interface IGeoConfig {
     mdObject?: VisualizationObject.IVisualizationObjectContent;
     selectedSegmentItem?: string;
     tooltipText?: VisualizationInput.IAttribute;
-    zoom?: number;
+    zoom?: number; // 0 > 22
```

Comment:
Can we leave comment `// in the 0-22 zoom range`?

Reply:
updated